packages/camp/package.py

```diff
@@ -80,11 +80,11 @@ def mpi_for_radiuss_projects(options, spec, env):
         mpi_exec_index = [
             index for index, entry in enumerate(options) if "MPIEXEC_EXECUTABLE" in entry
         ]
+        if len(mpi_exec_index) > 0:
+            del options[mpi_exec_index[0]]
         mpi_exec_flag_index = [
             index for index, entry in enumerate(options) if "MPIEXEC_NUMPROC_FLAG" in entry
         ]
-        if len(mpi_exec_index) > 0:
-            del options[mpi_exec_index[0]]
```

Comment:
This is flagged as a "fixup". Do we agree that this change is cosmetic?

Reply:
Not cosmetic - the index calculations cannot be grouped since after one entry is removed, the index for the next removal is off by one.

Reply:
OK "options" was the corner stone, I missed it.